README.md

```diff
@@ -1,7 +1,73 @@
 # SBC-ProductHub
+This Repo consists of 2 projects:
 
+1. **Metrics report** (Within application-status folder)
+
+2. **Release Notes** (Within release-notes folder)
 ## Apps
 
 ### Application Status
+An MVP Product [Metrics Report](www.metrics.bcregistry.gov.bc.ca)
+
+The app focuses on getting the metrics for all products developed by **Ministry of Citizen Services, Business Registry Branch**.
+
+The app displays the statistics of what the team has been working on during the last sprint so that the team can reflect on their team performance and keep track of the progress of their development. 
```

Comment:
would it better to talk about "teams" instead of "the team", e.g.:
The app displays statistics of what teams have been working on during the last sprint, so that teams can reflect on their performance and keep track of the development progress.